crates/client-api/src/routes/subscribe.rs

```diff
@@ -243,6 +247,7 @@ async fn ws_client_actor_inner(
                         // now we flush all the messages to the socket
                         ws.flush().await
                     };
+                    let send_all = also_poll(send_all, handle_queue.make_progress());
```

Comment:
```suggestion
                    // Flush the websocket while continuing to poll the `handle_queue`,
                    // to avoid deadlocks or delays due to enqueued futures holding resources.
                    let send_all = also_poll(send_all, handle_queue.make_progress());
```